src/shipping/package/app/initializer.ts

```diff
@@ -24,7 +24,8 @@ export class PackageServiceInitializer
     private static async initMongoDb(connection: string, collectionName: string) {
         try {
             var db = (await MongoClient.connect(connection)).db();
-            await db.command({ shardCollection: db.databaseName + '.' + collectionName, key: { tag: "hashed" } });
+            //await db.command({ shardCollection: db.databaseName + '.' + collectionName, key: { tag: "hashed" } });
+            await db.command({customAction: "CreateCollection", collection: db.databaseName + '.' + collectionName});
```

Comment:
Please remove the commented line, unless there's a reason to keep it, in which case we should explain that with another comment:

```suggestion
            await db.command({customAction: "CreateCollection", collection: db.databaseName + '.' + collectionName});
```